cmd/kubelet/app/options/options.go

```diff
@@ -87,14 +87,14 @@ func NewKubeletServer() *KubeletServer {
 			HostIPCSources:              kubetypes.AllSource,
 			HTTPCheckFrequency:          unversioned.Duration{20 * time.Second},
 			ImageMinimumGCAge:           unversioned.Duration{2 * time.Minute},
-			ImageGCHighThresholdPercent: 90,
-			ImageGCLowThresholdPercent:  80,
+			ImageGCHighThresholdPercent: 80,
+			ImageGCLowThresholdPercent:  70,
```

Comment:
I don't think we need to change the image GC threshold since this is based on the disk usage.